Lib/test/test__osx_support.py

```diff
@@ -7,74 +7,117 @@
 import stat
 import sys
 import unittest
-
 import test.support
-
 import _osx_support
 
+from unittest import mock
+from distutils.tests import support
+
+
 @unittest.skipUnless(sys.platform.startswith("darwin"), "requires OS X")
-class Test_OSXSupport(unittest.TestCase):
+class Test_OSXSupport(support.LoggingSilencer,
+                      unittest.TestCase):
 
     def setUp(self):
+        super().setUp()
         self.maxDiff = None
         self.prog_name = 'bogus_program_xxxx'
         self.temp_path_dir = os.path.abspath(os.getcwd())
         self.env = test.support.EnvironmentVarGuard()
         self.addCleanup(self.env.__exit__)
         for cv in ('CFLAGS', 'LDFLAGS', 'CPPFLAGS',
-                            'BASECFLAGS', 'BLDSHARED', 'LDSHARED', 'CC',
-                            'CXX', 'PY_CFLAGS', 'PY_LDFLAGS', 'PY_CPPFLAGS',
-                            'PY_CORE_CFLAGS'):
+                   'BASECFLAGS', 'BLDSHARED', 'LDSHARED', 'CC',
+                   'CXX', 'PY_CFLAGS', 'PY_LDFLAGS', 'PY_CPPFLAGS',
+                   'PY_CORE_CFLAGS'):
             if cv in self.env:
                 self.env.unset(cv)
 
+    def tearDown(self):
+        super().tearDown()
```

Comment:
Is it necessary to implement a method whose only job is to call the method in the superclass? If so, needs a comment explaining why.